test/fixtures/shielded_vm/main.tf

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// using default unshielded VM image for both client and server VMs
+module "unshielded-vm" {
```

Comment:
The integration tests are failing on creating the resource `module.unshielded-vm.module.server.google_compute_instance.forseti-server`during the test setup.

Errors
```
Error: Error creating Firewall: googleapi: Error 404: The resource 'projects/ci-forseti-serv-phrl/global/networks/default' was not found, notFound
       
         on ../../../modules/client/main.tf line 146, in resource "google_compute_firewall" "forseti-client-deny-all":
        146: resource "google_compute_firewall" "forseti-client-deny-all" {
       
       
       
       Error: Error creating Firewall: googleapi: Error 404: The resource 'projects/ci-forseti-serv-phrl/global/networks/default' was not found, notFound
       
         on ../../../modules/client/main.tf line 146, in resource "google_compute_firewall" "forseti-client-deny-all":
        146: resource "google_compute_firewall" "forseti-client-deny-all" {
       
       
       
       Error: Error creating Firewall: googleapi: Error 404: The resource 'projects/ci-forseti-serv-phrl/global/networks/default' was not found, notFound
       
         on ../../../modules/client/main.tf line 170, in resource "google_compute_firewall" "forseti-client-ssh-external":
        170: resource "google_compute_firewall" "forseti-client-ssh-external" {
       
       
       
       Error: Error creating Firewall: googleapi: Error 404: The resource 'projects/ci-forseti-serv-phrl/global/networks/default' was not found, notFound
       
         on ../../../modules/client/main.tf line 170, in resource "google_compute_firewall" "forseti-client-ssh-external":
        170: resource "google_compute_firewall" "forseti-client-ssh-external" {
       
       
       
       Error: Error creating Firewall: googleapi: Error 404: The resource 'projects/ci-forseti-serv-phrl/global/networks/default' was not found, notFound
       
         on ../../../modules/server/main.tf line 119, in resource "google_compute_firewall" "forseti-server-deny-all":
        119: resource "google_compute_firewall" "forseti-server-deny-all" {
       
       
       
       Error: Error creating Firewall: googleapi: Error 404: The resource 'projects/ci-forseti-serv-phrl/global/networks/default' was not found, notFound
       
         on ../../../modules/server/main.tf line 119, in resource "google_compute_firewall" "forseti-server-deny-all":
        119: resource "google_compute_firewall" "forseti-server-deny-all" {
       
       
       
       Error: Error creating Firewall: googleapi: Error 404: The resource 'projects/ci-forseti-serv-phrl/global/networks/default' was not found, notFound
       
         on ../../../modules/server/main.tf line 143, in resource "google_compute_firewall" "forseti-server-ssh-external":
        143: resource "google_compute_firewall" "forseti-server-ssh-external" {
       
       
       
       Error: Error creating Firewall: googleapi: Error 404: The resource 'projects/ci-forseti-serv-phrl/global/networks/default' was not found, notFound
       
         on ../../../modules/server/main.tf line 143, in resource "google_compute_firewall" "forseti-server-ssh-external":
        143: resource "google_compute_firewall" "forseti-server-ssh-external" {
       
       
       
       Error: Error creating Firewall: googleapi: Error 404: The resource 'projects/ci-forseti-serv-phrl/global/networks/default' was not found, notFound
       
         on ../../../modules/server/main.tf line 177, in resource "google_compute_firewall" "forseti-server-allow-grpc":
        177: resource "google_compute_firewall" "forseti-server-allow-grpc" {
       
       
       
       Error: Error creating Firewall: googleapi: Error 404: The resource 'projects/ci-forseti-serv-phrl/global/networks/default' was not found, notFound
       
         on ../../../modules/server/main.tf line 177, in resource "google_compute_firewall" "forseti-server-allow-grpc":
        177: resource "google_compute_firewall" "forseti-server-allow-grpc" {
       
       
       
       Error: Error creating instance: googleapi: Error 400: Invalid value for field 'resource.networkInterfaces[0].subnetwork': 'projects/ci-forseti-serv-phrl/regions/us-central1/subnetworks/default'. The referenced subnetwork resource cannot be found., invalid
       
         on ../../../modules/server/main.tf line 225, in resource "google_compute_instance" "forseti-server":
        225: resource "google_compute_instance" "forseti-server" {
       
       
       
       Error: Error creating instance: googleapi: Error 400: Invalid value for field 'resource.networkInterfaces[0].subnetwork': 'projects/ci-forseti-serv-phrl/regions/us-central1/subnetworks/default'. The referenced subnetwork resource cannot be found., invalid
       
         on ../../../modules/server/main.tf line 225, in resource "google_compute_instance" "forseti-server":
        225: resource "google_compute_instance" "forseti-server" {
```

Reply:
Fixed.